Traceback (most recent call last):
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\jupyter_cache\executors\utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\nbclient\client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\nbclient\util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\nbclient\util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "c:\users\usuario\anaconda3\envs\wintest\lib\asyncio\base_events.py", line 587, in run_until_complete
    return future.result()
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\nbclient\client.py", line 554, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\nbclient\client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "c:\users\usuario\anaconda3\envs\wintest\lib\site-packages\nbclient\client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from IPython.display import display, Latex, Math, HTML, IFrame
from IPython.display import Markdown as md
from sympy import *
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m<ipython-input-1-9f67bc6f6264>[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [1;32mfrom[0m [0mIPython[0m[1;33m.[0m[0mdisplay[0m [1;32mimport[0m [0mdisplay[0m[1;33m,[0m [0mLatex[0m[1;33m,[0m [0mMath[0m[1;33m,[0m [0mHTML[0m[1;33m,[0m [0mIFrame[0m[1;33m[0m[1;33m[0m[0m
[0;32m      2[0m [1;32mfrom[0m [0mIPython[0m[1;33m.[0m[0mdisplay[0m [1;32mimport[0m [0mMarkdown[0m [1;32mas[0m [0mmd[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 3[1;33m [1;32mfrom[0m [0msympy[0m [1;32mimport[0m [1;33m*[0m[1;33m[0m[1;33m[0m[0m
[0m
[1;31mModuleNotFoundError[0m: No module named 'sympy'
ModuleNotFoundError: No module named 'sympy'

